src/components/ViewResponses/index.js

```diff
@@ -1,25 +1,38 @@
-import { Button, Card, Input, Form, Typography } from "antd";
+import { Button, Input, Form, Typography } from "antd";
 import { useEffect, useState } from "react";
-import { getFormResponses } from "../../utils/nostr";
+import { getFormResponses, getUserKind0s } from "../../utils/nostr";
 import Analytics from "./Analytics";
 import { useParams } from "react-router";
+import Response from "./Responses";
 
-function ViewResponses(props) {
+function ViewResponses() {
   const [nsecState, setNsecState] = useState("");
   const [responses, setResponses] = useState([]);
+  const [userInfo, setUserInfo] = useState({});
   const { nsec } = useParams();
   const { Text } = Typography;
 
   useEffect(() => {
     if (nsec) {
       getResponses(nsec);
     }
-  }, [nsec]);
+  });
 
   async function getResponses(nsecInput) {
```

Comment:
can be separated out from this component

Reply:
where should we move it? just outside the component in the same file?